src/Analysis/Ast/Impl/Extensions/PythonTypeExtensions.cs

```diff
@@ -37,8 +37,5 @@ public static bool IsGeneric(this IPythonType value)
                 dst.Location = src.Location;
             }
         }
-
```

Comment:
Removing because was unused and does not check for declaration type and null check for name.